### ISSUE TITLE
User Login

### ISSUE THREAD

**Issue by TamaraHoock** [amosproj/amos2022ss04-digital-identity#5]
## User story
1. As a User (HR employee)
2. I want / need to login on DIdentity platform with my e-mail address and password 
3. So that I can access my account to manage the digital identities of the employees.

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
Backend: @jackDS008, @Jean28518
FrontEnd: @annikakrause, @valentinBraeutigam, @steve-237 @Idontker

**Comment by Idontker** [comment]
@TamaraHoock 
I think the register button in the acceptance criteria might be not fitting on the login page. 
> - Register button

I thought only admins or HR employees can register/invite new users. Therefore you need to be logged in before you can register new users. 
Did I miss something?
